sonic-ycabled/ycable/ycable_utilities/y_cable_helper.py

```diff
@@ -80,6 +97,19 @@ def format_mapping_identifier(string):
 
     return s
 
+# Find out the underneath physical port list by logical name
+
+
+def logical_port_name_to_physical_port_list(port_name):
+    if port_name.startswith("Ethernet"):
+        if y_cable_platform_sfputil.is_logical_port(port_name):
+            return y_cable_platform_sfputil.get_logical_to_physical(port_name)
+        else:
+            helper_logger.log_error("Invalid port '%s'" % port_name)
+            return None
+    else:
+        return [int(port_name)]
+
 
 def y_cable_wrapper_get_presence(physical_port):
```

Comment:
Actually I am thinking that this should not be used from sonic-xcvrd, mainly because if xcvrd is not running the global chassis/sfputil instances will not be present and can cause ycable daemon not get desired API result. 
However from code reusability point of view, we should move this to a common lib ex sonic-py-common where we can passin the global chassis/sfputil instance and get the result. If that's okay I will open a seperate PR for this.